vshard/router/init.lua

```diff
@@ -1026,11 +1099,13 @@ end
 --     though all replicasets were scanned.
 --
 local function router_partial_callrw(router, bucket_ids, func, args, opts, callback)
+    local master, conn
```

Comment:
Where are they used? Can't see in the code.

Reply:
Dead code, removed.